### DISCUSSION TITLE
Using (deep|any) sleep mode and timer based wake up

### DISCUSSION THREAD

**Discussion by CK4444** [HomeSpan/HomeSpan#408] [Ideas]
Hi, since the HomeSpan devices are powered all the time they consume quit a bit of juice. Since some information like temperature and humidity is not needed to be updated every 10sec or so it would be nice to have the opportunity to wake up every 5 minutes, pair, send data and fall asleep again. 

I am afraid, that HomeKit does not support this kind of behaviour ? But if - would be cool to have it...

Anyway - big thanks for providing this incredible library...
Christian

**Comment by HomeSpan** [comment]
The next release will include a dedicated class supporting ESP-NOW, which provides a good framework for deploying battery-operated remote sensors. All code and documentation is available in the dev branch -  See NOW.md under "docs" in the dev branch for full details.

**Comment by CK4444** [comment]
Thank you, great news. 
Is it possible to have esp8266 as remote sensor using ESP NOW or is this restricted to the ESP32?

**Comment by HomeSpan** [comment]
Based on what I've read, the 8266 supports ESP-NOW but HomeSpan itself will not compile on an 8266.  The new class being added to HomeSpan for ESP-NOW support would likely need to be modified to be able to operate on the 8266.  Note this class is very self-contained.  You would not have to port over all of HomeSpan, just the new class (it's called SpanPoint - to be released in next update).

**Reply by CK4444** [reply]
Thank you very much - there are a couple of ESP Now articles out there - I will try to  manage…
